README.md

```diff
@@ -11,23 +11,61 @@
 
 [![License](https://img.shields.io/badge/License-Apache_2.0-lightblue.svg)](https://opensource.org/licenses/Apache-2.0)
 
-
 This driver is an extension of the [Selenium](http://docs.seleniumhq.org/) C# client. It has 
 all the functionalities of the regular driver, but add Appium-specific methods on top of this.
 
-## Appium server compatibility for v5.x 
-
-In case you are using the latest beta client v5.x please be aware you will either have to upgrade your appium server to 2.x or add the base-path argument:
-`appium --base-path=/wd/hub`, due to a breaking change on the default server base path. <br/>
-Regardless, it's highly recommended you move to appium 2.x since appium 1.x is no longer maintained. <br/>
-For more details about how to migrate to 2.x, see the following link : [appium 2.x migrating](https://appium.github.io/appium/docs/en/2.0/guides/migrating-1-to-2/)
-
-## WinAppDriver Notice!
+ ## v5 Release Candidate 
+ 
+> [!IMPORTANT]
+> ### Appium server compatibility for v5.x
+> 
+> In case you are using the latest beta client v5.x please be aware you will either have to upgrade your appium server to 2.x or add the base-path argument:
+> `appium --base-path=/wd/hub`, due to a breaking change on the default server base path. <br/>
+> Regardless, moving to appium 2.x is highly recommended since appium 1.x is no longer maintained. <br/>
+> For more details about how to migrate to 2.x, see the following link : [appium 2.x migrating](https://appium.github.io/appium/docs/en/2.0/guides/migrating-1-to-2/)
+
+> [!CAUTION]
+> ### Deprecated Methods
+> 
+> - `MultiAction` and `TouchAction` are deprecated. Please use W3C WebDriver actions or mobile: extensions.
+> - `LaunchApp`, `CloseApp`, and `reset` are deprecated. 
```

Comment:
Maybe each deprecation message in the code has suggestions (e.g. use activate app instead), thus here probably can address something like "please read each deprecation message as an alternative method"

Reply:
That's why I added additional info section :
> App management: Please read [issue #15807](https://github.com/appium/appium/issues/15807) for more details